reflex-app/src/animations/SocialLineIcons/SocialLineIcons.js

```diff
@@ -0,0 +1,116 @@
+import React, { Fragment, Component, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const traceFb = keyframes`
+  0% {
+    stroke-dashoffset: 200;
+  }
+  75% {
+    stroke-dashoffset: 0;
+    fill: rgba(66,102,178,0.3);
+  }
+  100% {
+    stroke-dashoffset: 0;
+    fill: #4266b2;
+  }
+`
+
+const traceTw = keyframes`
+  0% {
+    stroke-dashoffset: 300;
+  }
+  75% {
+    stroke-dashoffset: 0;
+    fill: rgba(29,161,242,0.3);
+  }
+  100% {
+    stroke-dashoffset: 0;
+    fill: #1DA1F2;
+  }
+`
+
+const traceMd = keyframes`
+  0% {
+    stroke-dashoffset: 400;
+  }
+  75% {
+    stroke-dashoffset: 0;
+    fill: rgba(0,0,0,0.3);
+  }
+  100% {
+    stroke-dashoffset: 0;
+    fill: black;
+  }
+`
+
+const Wrapper = styled.div`
+  position: relative;
+`
+
+const Fb = styled.svg`
+  width: 24px;
+  height: 30px;
+  stroke: #4266b2;
+  stroke-width: 1.5;
+  fill: rgba(66,102,178,0.3);
+  stroke-dasharray: 200;
+  stroke-dashoffset: 200;
+  margin-right: 6px;
+
+  animation: ${props => props.hover && `${traceFb} 0.5s ease-in-out forwards`};
+`;
+
+const Tw = styled.svg`
+  width: 24px;
+  height: 30px;
+  stroke: #1DA1F2;
+  stroke-width: 1.5;
+  fill: rgba(29,161,242,0.3);
+  stroke-dasharray: 300;
+  stroke-dashoffset: 300;
+  margin-right: 6px;
+
+  animation: ${props => props.hover && `${traceTw} 0.5s ease-in-out forwards`};
+`;
+
+const Md = styled.svg`
+  width: 24px;
+  height: 30px;
+  stroke: black;
+  stroke-width: 1.5;
+  fill: rgba(0,0,0,0.3);
+  stroke-dasharray: 400;
+  stroke-dashoffset: 400;
+
+  animation: ${props => props.hover && `${traceMd} 0.5s ease-in-out forwards`};
```

Comment:
Looks like this animation is repeated for each icon. What are your thoughts on changing the delay for each one of them, such that we have a cascading effect e.g. Fb gets outlined first, Tw next, and then Md? Just for the sake of demo